java/test/jmri/jmrit/display/PreviewPanelTest.java

```diff
@@ -24,6 +24,19 @@ public void testCtor() {
         Assert.assertNotNull("PreviewPanel Constructor",p);
         JUnitUtil.dispose(df);
         JUnitUtil.dispose(editor);
+        
+        JUnitUtil.deregisterEditorManagerShutdownTask();
+        JUnitUtil.deregisterBlockManagerShutdownTask();
+    }
+    
+    @BeforeEach
+    public void setUp() {
+        JUnitUtil.setUp();
```

Comment:
In many tests, we have:
```
JUnitUtil.setUp();
JUnitUtil.resetInstanceManager();
```
It shouldn't be needed, but a couple of years ago, I discovered that this is in some cases are good to have. The reason is that when the InstanceManager resets the first time, some methods that disposes objects are using the new InstanceManager instead of the old one.

But maybe we should add a double call to `resetInstanceManager()` in `JUnitUtil.setUp()`. Maybe that would improve our tests, but I'm not sure how much effect it would have.

```
public static void setUp() {
    WAITFOR_DELAY_STEP = DEFAULT_WAITFOR_DELAY_STEP;
    WAITFOR_MAX_DELAY = DEFAULT_WAITFOR_MAX_DELAY;

    // all the setup for a MockInstanceManager applies
    setUpLoggingAndCommonProperties();

    // Do a minimal amount of de-novo setup
    resetInstanceManager();
    resetInstanceManager();    <--- Maybe we should add this to JUnitUtil.setUp()
}
```